core/src/main/java/com/datastax/oss/driver/internal/core/ProtocolFeature.java

```diff
@@ -31,5 +31,19 @@
    * @see <a href="https://issues.apache.org/jira/browse/CASSANDRA-10145">CASSANDRA-10145</a>
    */
   PER_REQUEST_KEYSPACE,
+
+  /**
+   * The ability to add a custom set of key/value pairs associated to each request.
+   *
+   * @see <a href="https://issues.apache.org/jira/browse/CASSANDRA-8553">CASSANDRA-8553</a>
+   */
+  CUSTOM_PAYLOAD,
+
+  /**
+   * The ability at the protocol level to detect when a prepared statement has had its columns altered.
```

Comment:
This line throws off the formatter as it's too long I think.

Reply:
Right, made this change last minute and didn't re-run `mvn install` thanks